common/repositories/base/base_character_corpses_repository.h

```diff
@@ -263,6 +275,10 @@ class BaseCharacterCorpsesRepository {
 		e.wc_7             = 0;
 		e.wc_8             = 0;
 		e.wc_9             = 0;
+		e.rez_time         = 0;
+		e.gm_exp            = 0;
+		e.killed_by         = 0;
+		e.rezzable         = 1;
```

Comment:
This looks like it was manually changed here, base repositories should always be generated from the script itself https://docs.eqemu.io/developer/repositories/#code-generation-repository-generator

Reply:
I ran the generator but then adjusted them because my fields were named `killedby` and `reztime` I know we like to have snake_case so I was honoring that.

Reply:
Base repo's should be immutable and only mutated by the script